string[] where_array
string[] what_array
string[] who_array
---
bool success
